interfaces/PA-App/src/app/personal-components/select-language/select-language.component.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { SelectLanguageComponent } from './select-language.component';
+
+describe('SelectLanguageComponent', () => {
+  let component: SelectLanguageComponent;
+  let fixture: ComponentFixture<SelectLanguageComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ SelectLanguageComponent ],
```

Comment:
I think adding the TranslationModule in this blunt way, would fix the test:
```suggestion:
      imports: [
        TranslateModule.forRoot(),
      ],
```